de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,115 @@
+    config-opts:
+      - -Dgtk_doc=false
+      - -Dintrospection=false
+      - -Dmanpage=false
+      - -Dvapi=false
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.4.tar.xz
+        sha256: 163d08d783be6d4ab9a979ceb5a4fecbc1d9660d3c34168c581301cd53912b20
+
+  - name: krb5
+    subdir: src
+    config-opts:
+      - --localstatedir=/var/lib
+      - --sbindir=${FLATPAK_DEST}/bin
+      - --disable-static
+      - --disable-rpath
+    cleanup:
+      - /bin
+      - /share/et
+      - /share/examples
+      - /share/man
+    sources:
+      - type: archive
+        url: https://github.com/krb5/krb5/archive/refs/tags/krb5-1.21.2-final.tar.gz
+        sha256: 5827fa39ead0e7f7b8a158799eb5d50b194129e81b378892caedcafe50c5c195
+      - type: shell
+        dest: src
+        commands:
+          - autoreconf -si
+
+  - name: dbus-run-session
+    buildsystem: autotools
+    cleanup: ['*']
+    sources:
+      - type: archive
+        url: https://gitlab.freedesktop.org/dbus/dbus/-/archive/dbus-1.14.10/dbus-dbus-1.14.10.tar.gz
+        sha256: 5fedcd6c17fbc62646af224e1aab11a334c463e7e7d89163a670ed24e7ab241c
+        x-checker-data:
+          type: anitya
+          project-id: 5356
+          url-template: https://gitlab.freedesktop.org/dbus/dbus/-/archive/dbus-$version/dbus-dbus-$version.tar.g00z
+
+  - name: dbus-python
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
+        sha256: ad67819308618b5069537be237f8e68ca1c7fcc95ee4a121fe6845b1418248f8
+        x-checker-data:
+          type: anitya
+          project-id: 402
+          url-template: https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$version.tar.gz
+
+  # external python modules
+  - python-modules.yml
+
+  - name: nagstamon
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py bdist --format tar
+      - tar -xvf dist/nagstamon*.tar --strip 2 -C /app
+      - install -D nagstamon.sh /app/bin/nagstamon
+      - mkdir -p /app/share/icons/hicolor/scalable/apps/
+      - patch -i nagstamon_svg_size.patch /app/share/pixmaps/nagstamon.svg
+      - install -Dm644 /app/share/pixmaps/nagstamon.svg /app/share/icons/hicolor/scalable/apps/
+      - install -Dm644 nagstamon.appdata.xml -t /app/share/metainfo
+    sources:
+      - type: archive
+        url: https://github.com/HenriWahl/Nagstamon/archive/refs/tags/v3.14.0.tar.gz
+        sha256: f51c50feb7efa328d252c638b6602405d550a99622dd7e8f07341814879e9f30
+      - type: file
+        url: https://raw.githubusercontent.com/HenriWahl/Nagstamon/master/Nagstamon/resources/nagstamon.appdata.xml
+        sha256: c9db793fa04bf20e28fb38535751e49640bd95d7d940bf2993bbd19c1ee0691a
+      - type: file
+        path: nagstamon.sh
+      - type: file
```

Comment:
this should by `type: patch` and patch the actual file in the source. and remove line 101.

Reply:
What exactly in this context?

The SVG is patched in the master branch but not yet in the current stable 3.14.0, that's what this patch is for.

Reply:
just switch to `type: patch` and remove line 100.

Reply:
@hfiguiere - OK, but what do you mean? Line 111 or 113? And why? Thanks for your patience so far but sorry for the stupid questions -I simply do not understand. Shouldnt' the SVG be patched? And isn't the patch included here in the repo?

Reply:
it's in the documentation

![image](https://github.com/flathub/flathub/assets/114441/abed6fb8-6839-40eb-8997-75f321066fe4)

I mean line 100 to remove. I was already pretty clear.

See the other comment

![image](https://github.com/flathub/flathub/assets/114441/b10b4c9e-9f52-4cef-ae18-437f899ca041)

tl;dr if you run the `patch` command you are likley doing it wrong.

Reply:
might be line 101 but the suggestion was there.

Reply:
Applied it.

Reply:
still not done

Reply:
Commited right now.